filebeat/module/kafka/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: kafka
+  # All logs
+  log:
+    enabled: true
+
+    # Set custom paths for Kafka. If left empty,
+    # Filebeat will look under /opt.
+    #var.kafka_home:
```

Comment:
I think we can put the default in the commented line here, so it's clear for the user how to edit. I.e.:

```
      #var.kafka_home: /opt/kafka*
```

Btw, it's `/opt/kafka*`, not just `/opt`, right?